src/org/ods/component/Context.groovy

```diff
@@ -74,6 +74,7 @@ class Context implements IContext {
         config.buildUrl = script.env.BUILD_URL
         config.buildTag = script.env.BUILD_TAG
         config.buildTime = new Date()
+        config.dockerRegistry = script.env.DOCKER_REGISTRY
```

Comment:
Would prefer to remove Docker since OpenShift serves a registry for OCI-compliant container images.

Reply:
if we change this we need to change it in ods-core etx.. i suggest to leave for now